### ISSUE TITLE
For March 13th (beta), T&Cs

### ISSUE THREAD

**Issue by MollieCourtenay** [healthlocker/healthlocker#114]
## Acceptance Criteria  

+ [x] as a user I can click on a link in the footer to load the T&Cs.

## Terms of Service
This statement governs the terms of service between you, the user and Healthlocker. By using Healthlocker you agree to the following statement: 

**Privacy**
Your privacy is important to us, for this reason we have designed a privacy statement which will outline how we protect your privacy. We ask you to read this statement so that you can make an informed decision regarding creating your account.

**Sharing of Information**
Healthlocker is designed so you, as the user, have complete control over who you add to your friend circle and what information you share with each person.   

**Safety**
Whilst Healthlocker sits on a secure platform, and the chance of a data breach is very unlikely, we cannot absolutely guarantee the safety of Healthlocker. As the user you will need to make sure your login in details are kept confidential and not shared with anyone. You will also need to be aware of your surroundings, and we advise you only login to your account when you are in a private space.  
You as the user will need to ensure that you only add people to your circle of friends that you know, and that you want to be part of your ongoing healthcare. 

**Setting up your account**
When setting up your account you must agree to the following information:

- that you will not provide false information when setting up your account;
- that you will keep all your information updated
- that you will not share your login details with anyone
- that you will respect other users. 

**Terminations**
If you violate the spirit of this agreement Healthlocker can suspend or terminate your account. This decision will be sent to you via email or you will receive a notification the next time you try to enter your account.
You do have the option of deleting your account. At which time, all information on your account will not be accessible to any user in your circle of friends but will be retained in case you would like to reactive your account at a later date.  

**Other Applications**
Healthlocker does contain links to other application which are not owned or controlled by South London and Maudsley NHS Foundation Trust. SLaM cannot take responsibility for the content or general practices of these applications.  

**Changes to Agreement**
This agreement may be updated from time to time. In this event, Healthlocker will always notify you of the changes and ask for your ongoing consent.

**Comment by katbow** [comment]
@MollieCourtenay would this be the separate page containing full T&Cs we discussed earlier (when I was completing sign up)?

**Comment by MollieCourtenay** [comment]
@katbow yes

**Comment by katbow** [comment]
@MollieCourtenay any word on content yet? The pages are all set up and ready to go, just need to put the content in once we have it!

**Comment by MollieCourtenay** [comment]
@katbow  content above in from SLaM

**Comment by katbow** [comment]
Sweet! Thanks

**Comment by MollieCourtenay** [comment]
Hello - I am very sorry to now give you edits. I had trusted this would be ready, but I've had to make some corrections...

+ [ ] privacy paragraph  -  missing a full stop. account.

+ [ ] privacy paragraph  -  [privacy statement](https://www.healthlocker.uk/pages/privacy)

+ [ ] **Sharing of Information** 
Healthlocker is designed so you, as the user, have complete control over who you connect with and what information you share with them.   

+ [ ]  **Security** (last sentence)
You as the user will need to ensure that you only connect to other Healthlocker users that you know, and that you want to be part of your ongoing healthcare. 

+ [ ] **Terminations**
You do have the option of deleting your account. At which time, all information you have previously shared with others will still be available to them. Your account will be safely retained in case you would like to reactivate your account at a later date.

+ [ ] **Other Applications**
Healthlocker does contain links to other applications which are not owned or controlled by South London and Maudsley NHS Foundation Trust.
